reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -516,6 +517,27 @@ public static void disableContextLossTracking() {
 		DETECT_CONTEXT_LOSS = false;
 	}
 
+	/**
+	 * Globally enables the {@link Context} loss detection in operators like
```

Comment:
These javadocs seem to be a copy/paste of the `ContextLossTracking` feature, *not* `StatsTracking`

Reply:
Fixed 🙏